packages/commonwealth/server/controllers/server_threads_methods/get_threads_by_id.ts

```diff
@@ -92,6 +93,44 @@ export async function __getThreadsById(
           },
         ],
       },
+      {
```

Comment:
Need a query plan for this query too. To get the raw query you can use `logging: console.log`

Reply:
before:
https://explain.dalibo.com/plan/4fbe28b81957e4cc

after:
https://explain.dalibo.com/plan/86eace9g47319162